src/main/java/org/broadinstitute/hellbender/tools/walkers/bqsr/BaseRecalibrator.java

```diff
@@ -147,12 +147,13 @@ public void onTraversalStart() {
         referenceDataSource = ReferenceDataSource.of(referenceArguments.getReferenceFile());
     }
 
-    private void assertNoSOLiDReadGroups(final SAMFileHeader readsHeader) {
-        if (readsHeader.getReadGroups().stream().anyMatch(rg -> NGSPlatform.fromReadGroupPL(rg.getPlatform()) ==  NGSPlatform.SOLID)){
-            throw new UserException.BadInput("BQSR does not support SOLiD data but found read group from SOLiD platform.");
+    private void assertOnlyIlluminaReadGroups(final SAMFileHeader readsHeader) {
+        if (readsHeader.getReadGroups().stream().anyMatch(rg -> NGSPlatform.fromReadGroupPL(rg.getPlatform()) !=  NGSPlatform.ILLUMINA)){
+            throw new UserException.BadInput("BQSR only supports ILLUMINA.");
```

Comment:
Same comment as above.  Maybe make this a public method that both classes can use?